packages/contracts-bedrock/test/invariants/CrossDomainMessenger.t.sol

```diff
@@ -108,6 +108,13 @@ contract XDM_MinGasLimits is Bridge_Initializer {
         // Don't allow the estimation address to be the sender
         excludeSender(Constants.ESTIMATION_ADDRESS);
 
+        // Don't allow the predeploys to be the senders
+        uint160 prefix = uint160(0x420) << 148;
+        for (uint256 i = 0; i < 2048; i++) {
+            address addr = address(prefix | uint160(i));
```

Comment:
some predeploys (ie. the bridges) do make withdrawals

Reply:
This contract is testing the L1 portion so I don't believe we need to care about the L2 predeploys. One problem is that we don't use the foundry concept of multiple forks (ie different state spaces) so all of the contracts live in the same state which I believe is the reason for this flake

Reply:
oh... I see. OK.